bot/exts/valentines/valentine_zodiac.py

```diff
@@ -44,14 +44,16 @@ def generate_invalidname_embed(self, zodiac: str) -> discord.Embed:
         """Returns error embed."""
         embed = discord.Embed()
         embed.color = Colours.soft_red
-        error_comp = "\n".join(
-            f"`{i}` {name}: {zodiac['start_at'].strftime('%B `%d`')} - {zodiac['end_at'].strftime('%B `%d`')}"
-            for i, (name, zodiac) in enumerate(sorted(self.zodiac_fact.items()), start=1)
-        )
-        embed.description = (
-            f"**{zodiac}** is not a valid zodiac sign, here is the list of valid zodiac signs.\n"
-            f"{error_comp}"
-        )
+        error_msg = f"**{zodiac}** is not a valid zodiac sign, here is the list of valid zodiac signs.\n"
+        valid_zodiac_name = ''
+        for name in sorted(self.zodiac_fact.keys()):
+            if name == "Leo":
+                valid_zodiac_name += f'{name}\n '
+            elif name == 'Virgo':
+                valid_zodiac_name += name
+            else:
+                valid_zodiac_name += f'{name}, '
```

Comment:
This seems a bit over engineered imo.
```suggestion
        valid_zodiac_name = ", ".join(self.zodiac_fact).replace("Leo, ", "Leo,\n")
```
Quick and dirty, given Leo is always written this way, and contains a space. So if it somehow ends up at the end it breaks.

So a better solution might just be to go by index
```py
", ".join(names[ :len(self.zodiac_fact) // 2]) + ",\n" + ", ".join(names[len(self.zodiac_fact) // 2: ])
```

Reply:
This can be written a bit cleaner with 2 variables for first and last part and add them afterwards.